charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```
@@ -12,7 +12,78 @@ spec:
     spec:
       containers:
         - name: repo-creator
-          image: quay.io/redhat-ai-dev/helm-chart-application-gitops:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #!/usr/bin/env bash
+              cd /tmp || exit
+              wget https://github.com/cli/cli/releases/download/v2.62.0/gh_2.62.0_linux_amd64.tar.gz
+              gunzip gh_2.62.0_linux_amd64.tar.gz
+              tar xf gh_2.62.0_linux_amd64.tar
```

```suggestion
              tar xzf gh_2.62.0_linux_amd64.tar
```
I think less command verbosity would be fine here, and just let tar unzip also

---

yep

---

fyi rather than `tar xzf gh_2.62.0_linux_amd64.tar` it needs to be `tar xzf gh_2.62.0_linux_amd64.tar.gz`